cs.html

```diff
@@ -96,13 +96,16 @@ <h3>Interface Definition</h3>
         <pre class="idl">
 interface QuicTransportBase {
     readonly        attribute QuicTransportState    state;
+    readonly        attribute unsigned short        maxDatagramSize;
     void                  stop (QuicTransportStopInfo stopInfo);
     QuicBidirectionalStream         createBidirectionalStream ();
     QuicSendStream                  createSendStream (optional QuicStreamParameters parameters);
+    QuicSendDatagramResult          sendDatagram (Uint8Array data);
```

Comment:
QuicSendDatagramResult should be defined somewhere.

Reply:
I thought I had defined it, but I don't see any record of it, even before I moved this to cs.html.  I must have forgotten to do it.  Good catch.  I'll add it.

Reply:
FYI, it's just a boolean now